src/main/java/edu/harvard/h2ms/seeders/EventSeeder.java

```diff
@@ -0,0 +1,126 @@
+package edu.harvard.h2ms.seeders;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import edu.harvard.h2ms.domain.core.*;
+import edu.harvard.h2ms.repository.*;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.event.ContextRefreshedEvent;
+import org.springframework.context.event.EventListener;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+import javax.persistence.Transient;
+import java.util.*;
+
+import static java.lang.Boolean.FALSE;
+import static java.lang.Boolean.TRUE;
+import static java.util.Arrays.asList;
+
+@Component
+public class EventSeeder {
+
+    @JsonIgnore
+    @Transient
+    @Autowired
+    private PasswordEncoder passwordEncoder;
+
+    private EventRepository eventRepository;
+    private AnswerRepository answerRepository;
+    private UserRepository userRepository;
+    private EventTemplateRepository eventTemplateRepository;
+    private QuestionRepository questionRepository;
+
+    @Autowired
+    public EventSeeder(EventRepository eventRepository, AnswerRepository answerRepository, UserRepository userRepository,
+                       EventTemplateRepository eventTemplateRepository, QuestionRepository questionRepository) {
+        this.eventRepository = eventRepository;
+        this.answerRepository = answerRepository;
+        this.userRepository = userRepository;
+        this.eventTemplateRepository = eventTemplateRepository;
+        this.questionRepository = questionRepository;
+    }
+
+    @EventListener
+    public void seed(ContextRefreshedEvent event) {
+        seedEventTable();
+    }
+
+    private void seedEventTable() {
+
+        List<List<String>> records = asList(
+                asList("jane", "smith", "jane.smith@email.com", "Dietician", "Y"),
+                asList("john", "doe", "john.doe@email.com", "Surgical Tech", "N"),
+                asList("luke", "skywalker", "luke.walker@email.com", "Other", "Y"),
+                asList("jon", "snow", "jon.snow@email.com", "Other", "Y"),
+                asList("arya", "stark", "arya.stark@email.com", "Surgical Tech", "Y"),
+                asList("joffrey", "baratheon", "joffrey.baratheon@email.com", "Other", "N")
```

Comment:
The main point of [seeding data](https://en.wikipedia.org/wiki/Database_seeding) is to seed things that are usable in the app when it gets deployed, like an administrative account or the handwashing template.  The stuff here is dummy data or test fixtures.  It shouldn't always get deployed.  Could you use `@Profile("test")` here so it doesn't always happen? Or we could write a script to write some test data.

Reply:
We should let it for now and update later else when these UI guys hit they won’t have results. But can remove if you want and adjust tests

Reply:
I guess you can but you could use the actual handwashing template instead of seeding a new one